app/controllers/skills_controller.rb

```diff
@@ -7,9 +7,22 @@ def index
     render json: fetch_entries, each_serializer: SkillSerializer, include: '*'
   end
 
+  def new_to_person
```

Comment:
Kannst du das noch zu `unvalued` oder so umbenennen, new ist ein keyword in Rails und hat bei mir recht für Verwirrung gesorgt